website/vercel.json

```
@@ -38,6 +38,16 @@
     }
   ],
   "redirects": [
+    {
+      "source": "/docs/cloud/cost-management",
+      "destination": "/docs/cloud/about-cloud/overview",
```

this is also taking me to a page not found. maybe we mean this:
```suggestion
      "destination": "/docs/cloud/about-cloud/dbt-cloud-features",
```